projects.yaml

```diff
@@ -1672,6 +1673,14 @@ projects:
   pypi_id: mkdocs-auto-refresh-build-pages
   labels: [plugin]
   category: site-management
+- name: Mkdocs Quiz
+  mkdocs_plugin: mkdocs_quizz
```

Comment:
Your entrypoint specifies `mkdocs_quiz` (single z) as name: https://github.com/bdallard/mkdocs_quiz/blob/main/setup.py#L28C14-L28C25. Could you fix either of this line or your entrypoint :smile:? Also I see that for some reason, `mkdocs-quiz` and `mkdocs-quiz-plugin` were already taken on PyPI, hence maybe why you used `mkdocs-quizz` (which seems like the obvious thing to do since it's the right spelling?). So, you might want to use "quizz" (two z) everywhere (including for your repository name)? Just a suggestion of course (this could reduce confusion for your users).

Reply:
Edit again, looks like American English uses quizz while British English uses quiz :shrug:

Reply:
Yees indeed you got it... Initially I used quiz without the two z's but the name was already taken on pypi that's why I did it at the last minute... sorry for the confusion😅